ui/tests/e2e/extensions.spec.js

```diff
@@ -43,11 +43,18 @@ test.describe('Extensions Section Tests', () => {
     await expect(performanceEnableButton).toBeEnabled();
   });
 
-  test('Verify Kanvas Details', async ({ page }) => {
+  test('Verify Kanvas Details', async ({ page, context }) => {
     await expect(page.getByTestId('kanvas-signup-heading')).toBeVisible();
     const kanvasDetailsButton = page.getByTestId('kanvas-signup-btn');
     await expect(kanvasDetailsButton).toBeVisible();
-    await expect(kanvasDetailsButton).toBeDisabled();
+    if (await kanvasDetailsButton.isEnabled()) {
+      const [docsPage] = await Promise.all([
```

Comment:
ensure the error handling is proper here, promise.all is generally not safe. If either of the event fails the test case will fail.

Reply:
I think we are fine here in playwright regarding this pattern as we have two events that are tightly coupled and if one fails (i/e the click doesn't open the page) thats a valid reason for the test to fail anyways.